### PR TITLE
Fix/aim multipage select

### PR DESCRIPTION
**Pull request by OzgeYurtsever**

### CODE REVIEW

**Review by emelalkim — APPROVED** [review]

File: src/components/annotationSearch/AnnotationTable.jsx [hunk 1]
```diff
@@ -20,13 +20,14 @@ import {
   annotationsLoadingError,
   updateSearchTableIndex,
   setSeriesData,
+  storeAimSelection
 } from "../annotationsList/action";
 import { formatDate } from "../flexView/helperMethods";
 import { getSeries, getSignificantSeries } from "../../services/seriesServices";
 import SelectSerieModal from "../annotationsList/selectSerieModal";
-import { isSupportedModality } from "../../Utils/aid.js";
+import { isSupportedModality, findSelectedCheckboxesMap } from "../../Utils/aid.js";
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/AnnotationTable.jsx:28]: done

File: src/components/annotationSearch/AnnotationTable.jsx [hunk 2]
```diff
@@ -20,13 +20,14 @@ import {
...
 import { COMP_MODALITIES as compModality, teachingFileTempCode } from "../../constants.js";
-const defaultPageSize = 200;
+const defaultPageSize = 10;
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/AnnotationTable.jsx:30]: done

File: src/components/annotationSearch/AnnotationTable.jsx [hunk 3]
```diff
@@ -337,13 +356,14 @@ function AnnotationTable(props) {
 
   useEffect(() => {
     preparePageData(props.data, defaultPageSize, props.searchTableIndex);
-  }, [props.pid, props.data]);
+
+  }, [props.pid, props.data, props.multiFrameAimJumpData]);
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/AnnotationTable.jsx:360]: done

File: src/components/annotationSearch/AnnotationTable.jsx [hunk 4]
```diff
@@ -337,13 +356,14 @@ function AnnotationTable(props) {
...
 
   useEffect(() => {
     if (props.data.length <= defaultPageSize * props.searchTableIndex) {
       preparePageData(props.data, defaultPageSize, props.searchTableIndex);
     }
-  }, [props.noOfRows, props.data, props.searchTableIndex]);
+  }, [props.noOfRows, props.data, props.searchTableIndex, props.multiFrameAimJumpData]);
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/AnnotationTable.jsx:366]: done

File: src/components/annotationSearch/index.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from "react";
 import { connect } from "react-redux";
 import { toast } from "react-toastify";
 import PropagateLoader from "react-spinners/PropagateLoader";
-import _ from "lodash";
+import _, { set } from "lodash";
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/index.jsx:5]: done

File: src/components/annotationSearch/index.jsx [hunk 6]
```diff
@@ -670,10 +673,12 @@ const AnnotationSearch = (props) => {
     if (Object.keys(filters).length) body["filter"] = newFilters;
     searchAnnotations(body, bm)
       .then((res) => {
+        console.log(" ----> in resolve search");
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/annotationSearch/index.jsx:676]: done

**Pull request merged**